scripts/configure

```diff
@@ -17,7 +17,7 @@ default_seed_repo = "https://github.com/khuedoan/homelab"
 default_domain = "khuedoan.com"
 default_timezone = "Asia/Ho_Chi_Minh"
 default_terraform_workspace = "khuedoan"
-
+default_dhcp_server = "dhcp_proxy: false"
```

Comment:
The configure script is intended to update basic configuration that _most_ people will need to update, if we enable DHCP proxy by default, we can add a link to the above documentation in prerequisites instead of a new option in the configure script (to avoid adding too many feature to the script):

https://github.com/khuedoan/homelab/blob/532199585282c26943b61c82461b28b4fe47327e/docs/installation/production/prerequisites.md?plain=1#L50-L53

Similar to single node cluster adjustment for Longhorn:

https://github.com/khuedoan/homelab/blob/532199585282c26943b61c82461b28b4fe47327e/docs/installation/production/prerequisites.md?plain=1#L43

Reply:
Agreed, will remove. This could also be an air-gap guide as a future todo